include/sys/pci.h

```diff
@@ -106,6 +106,12 @@ static inline void pci_write_config(device_t *device, unsigned reg,
   PCI_DRIVER(device)->pci_bus.write_config(device, reg, size, value);
 }
 
+/*! \brief Enable the bus master mode. */
+static inline void pci_enable_bus_master(device_t *device) {
```

Comment:
This should belong to PCI bus methods interface as in: http://bxr.su/FreeBSD/sys/dev/pci/pci_if.m#110

Reply:
Done.